app/navigation/IndividualProtectorNavigator.scala

```diff
@@ -74,13 +74,13 @@ class IndividualProtectorNavigator @Inject()() extends Navigator {
       .getOrElse(controllers.routes.SessionExpiredController.onPageLoad())
   }
 
-//  def checkDetailsRoute(answers: UserAnswers): Call = {
-//    answers.get(IndexPage) match {
-//      case None => controllers.routes.SessionExpiredController.onPageLoad()
-//      case Some(x) =>
-//        controllers.individual.amend.routes.CheckDetailsController.renderFromUserAnswers(x)
-//    }
-//  }
+  def checkDetailsRoute(answers: UserAnswers): Call = {
```

Comment:
Have tests been added for amend journey navigation in IndividualProtectorNavigatorSpec?